openeo/rest/__init__.py

```diff
@@ -2,6 +2,9 @@
 
 from openeo import BaseOpenEoException
 
+# TODO: get from config file
+DEFAULT_DOWNLOAD_CHUNK_SIZE = 10000000  # 10MB
```

Comment:
FYI in python you can make large numbers more readable with `_`
```suggestion
DEFAULT_DOWNLOAD_CHUNK_SIZE = 10_000_000  # 10MB
```